### PR TITLE
Limit format/type allowed for tex{Sub}Image*D with DOM elements

### PR DESCRIPTION
**Pull request by zhenyao**
Per discussion in today's working group meeting

### CODE REVIEW

**Comment by zhenyao** [comment]
@kenrussell @jdashg

File: specs/latest/2.0/index.html [hunk 1]
```diff
@@ -1366,7 +1366,38 @@ <h2 class="no-toc">Table of contents</h2>
...
+        <tr><td>RGBA8</td><td>RGBA</td><td>UNSIGNED_BYTE</td></tr>
+        <tr><td>SRGB8_ALPHA8</td><td>RGBA</td><td>UNSIGNED_BYTE</td></tr>
+        <tr><td>RGB5_A1</td><td>RGBA</td><td>UNSIGNED_BYTE<br>UNSIGNED_SHORT_5_5_5_1</td></tr>
+        <tr><td>RGBA4444</td><td>RGBA</td><td>UNSIGNED_BYTE<br>UNSIGNED_SHORT_4_4_4_4</td></tr>
+        <tr><td>RGBA16F</td><td>RGBA</td><td>HALF_FLOAT<br>FLOAT</td></tr>
+        <tr><td>RGBA32F</td><td>RGBA</td><td>FLOAT</td></tr>
+        <tr><td>RGBA8UI</td><td>RGBA_INTEGER</td><td>UNSIGNED_BYTE</td></tr>
+        </table>
+        <div class="note">When the data source is a DOM element (<code>Image</code>, <code>Canvas</code>, or <code>Video</code>), commonly each channel's representation is unsigned integer type of at least 8 bits. Converting such representation to signed integers or unsigned integers with more bits is not clearly defined. For example, when converting RGBA8 to RGBA16UI,  it is unclear whether or not the intension is to scale up values to the full range of 16-bit unsigned integer. Therefore, only converting to unsigned integer of at most 8 bits, half float, or float is allowed.</div>
```
Comment (kenrussell) [inline specs/latest/2.0/index.html]: Minor grammatical / spelling corrections:
"is unsigned integer type" -> "is an unsigned integer type"
"intension" -> "intention"
"range of 16-bit" -> "range of a 16-bit"

**Comment by kenrussell** [comment]
Looks great per today's working group discussion. Please merge yourself after making the above small corrections. Thanks.

File: specs/latest/2.0/index.html (hunk 1 above)
Comment (zhenyao) [inline specs/latest/2.0/index.html]: Done

**Pull request merged**